lib/generators/component.js

```diff
@@ -12,19 +12,25 @@ export function generateComponent(name, options, config) {
   ensureModuleExists(moduleName);
 
   _generate('component', moduleName, entityName, options, config);
-  _generateTest('component', moduleName, entityName, config);
+
+  if(config.generateComponentTests) {
+    _generateTest('component', moduleName, entityName, config);
+  }
   if (config.storybook) {
     generateStorybook(name, options, config);
   }
 }
 
-export function destroyComponent(name) {
+export function destroyComponent(name, options, config) {
   let [moduleName, entityName] = name.split(':');
 
   ensureModuleNameProvided(name);
   ensureModuleExists(moduleName);
 
   removeFile(getOutputPath('component', entityName, moduleName));
   destroyStorybook(name);
-  removeFile(getTestOutputPath('component', entityName, moduleName));
+
+  if(config.generateComponentTests) {
```

Comment:
not sure if the conditional is needed. I think we should always delete everything related to the component when removing.